src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -337,10 +294,10 @@ function PaymentMethodList({
 
     return (
         <>
-            <View style={[style, {minHeight: (filteredPaymentMethods.length + (shouldShowAddBankAccount && 1)) * variables.optionRowHeight}]}>
+            <View style={[style, {minHeight: (filteredPaymentMethods.length + (shouldShowAddBankAccount ? 1 : 0)) * variables.optionRowHeight}]}>
                 <FlashList
                     estimatedItemSize={variables.optionRowHeight}
-                    data={filteredPaymentMethods}
+                    data={filteredPaymentMethods as PaymentMethodItem[]}
```

Comment:
This change should not be needed

Reply:
Yup, we should avoid casts (`{something} as {something}`) at a high cost